woocommerce/woocommerce
# Remove test publishing

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





1. Swap the HPOS and non-HPOS testing workflows.  HPOS is the default, so it should be for our tests too.
2. Disable the publishing of daily test results to the dashboard for non HPOS runs (they aren't being displayed there)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Actions > Smoke test daily
3. Run the workflow using this branch and make sure there's no issues
4. Go to Actions > Run daily tests in an environment with HPOS enabled
5. Run this workflow using this branch and make sure there's no issues

Here's a couple if you don't want to wait:

- [HPOS disabled](https://github.com/woocommerce/woocommerce/actions/runs/7891861581)
- [Normal daily](https://github.com/woocommerce/woocommerce/actions/runs/7891865540)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
